angular-rails-app/app/assets/javascripts/property/property.controller.js

```diff
@@ -7,6 +7,41 @@ PropertyController.$inject = ['PropertyService','property'];
 
 function PropertyController(PropertyService, property) {
   var vm = this;  
-  vm.data = property.data;
+  vm.property = property.data;
+  vm.itemsPerPage = 10;
+
+  vm.tables = {
+    'complaint311s' : new Table("complaint311s"),
+    'dobPermits' : new Table("dob_permits"),
+    'dobViolations' : new Table("dob_violations"),
+    'hpdComplaints' : new Table("hpd_complaints"),
+    'hpdViolations' : new Table("hpd_violations"),
+    'litigations' : new Table("litigations")
+  }
+
+  function Table(urlResource) {
+    this.data = [];
+    this.urlResource = urlResource;
+    this.total = undefined;
+    this.page = 1;
+  }
```

Comment:
Nit: `Table` can be at the top level, instead of buried inside the PropertyController constructor.